ui-tests/src/test/java/org/openmrs/reference/page/RegistrationPage.java

```diff
@@ -166,6 +167,7 @@ public void clickOnContactInfo() throws InterruptedException {
     }
 
     public void clickOnPhoneNumber() throws InterruptedException {
+    	waiter.until(ExpectedConditions.visibilityOfElementLocated(PHONE_NUMBER_LABEL));
```

Comment:
Indention here! Correspond to the existing code indention.

Reply:
Oh sorry i forgot that, Fixed that